### PR TITLE
All in one errors

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/lib/installation/widgets/overview.rb [hunk 1]
```diff
@@ -50,19 +54,38 @@ def label
...
             { "simple_mode" => true }
           ])
-        d["label_proposal"]
+        if d["warning"] && !d["warning"].empty? && d["warning_level"] != :notice
+          Yast::Popup.LongError(
+            "Problem found when proposing %{client}:<br>" \
+            "Severity: %{severity}<br>" \
+            "Message: %{message}" %
+            ({
+              client: proposal_client,
```
Comment (mvidner) [inline src/lib/installation/widgets/overview.rb]: `proposal_client` is an implementation detail so we should not show it to the user. `label` is human readable, and that also means we should memoize it like we do for `items`

Comment (jreidinger) [inline src/lib/installation/widgets/overview.rb]: good idea, I do not like it before.

File: src/lib/installation/clients/inst_casp_overview.rb [hunk 2]
```diff
@@ -66,6 +74,15 @@ def run
           # do not store stuff when just redrawing
           skip_store_for: [:redraw]
         )
+        blocker = blocking_widgets.find(&:blocking?)
+        if blocker
+          # %title stands for title of problematic part
```
Comment (mvidner) [inline src/lib/installation/clients/inst_casp_overview.rb]: Just use `%s` when there is only one string to substitute? Less room for errors both for us and the translator.

> %s is a heading of a problematic section, like "Partitioning" or "Network"

Comment (jreidinger) [inline src/lib/installation/clients/inst_casp_overview.rb]: ok

File: src/lib/installation/clients/inst_casp_overview.rb [hunk 3]
```diff
@@ -66,6 +74,15 @@ def run
...
+          Yast::Popup.Error(
+            _("%{title} blocks installation. Please solve problem here before proceeding.") %
```
Comment (mvidner) [inline src/lib/installation/clients/inst_casp_overview.rb]: > %s blocks **the** installation. Please resolve **the** problem **there** before proceeding.

**Pull request merged**